docs/semgrep-supply-chain/getting-started.md

```diff
@@ -111,11 +111,11 @@ Semgrep Supply Chain supports the scanning of monorepos. As outlined in [Project
 
 ## Block pull or merge requests
 
-Semgrep Supply Chain versions **v0.122.0** and earlier automatically blocked pull/merge requests if it discovered reachable findings in the code, but later versions do not do this. You can, however, configure Semgrep Supply Chain to block on pull request scans that detect reachable findings in direct dependencies with high or critical severity.
+Semgrep Supply Chain versions **v0.122.0** and earlier automatically blocked pull/merge request scans if it discovered reachable findings in the code, but later versions do not do this. You can, however, configure Semgrep Supply Chain to block on scans that detect reachable findings in direct dependencies with high or critical severity.
```

Comment:
```suggestion
Semgrep Supply Chain versions **v0.122.0** and earlier automatically blocked pull/merge requests and scans if it discovered reachable findings in the code, but later versions do not do this. You can, however, configure Semgrep Supply Chain to block on scans that detect reachable findings in direct dependencies with high or critical severity and where there is an upgrade available.
```

I don't know if the first change is helpful or not (I think it is since it blocks both?), but I think the [second caveat is important to know](https://github.com/semgrep/semgrep-app/pull/14645).

Reply:
So for the first bit - I didn't necessarily want to fully change the topic of this over to scan blocking, since the section is called "Block pull or merge requests". I'm open to whether we want to change that title too, but since we still generally focus on PR/MRs for blocking, I hadn't done it myself. What do you think? Should we change it entirely?

For the second bit, absolutely! Not sure why it wasn't mentioned here, I think this even came up when I talked to Swati (who changed the toggle name) about this and we thought it could be useful to add. Good callout!

Reply:
Resolved in Slack :)